core/platform/datastore/gae_datastore_services.py

```diff
@@ -70,6 +70,20 @@ def get_multi(keys):
     return ndb.get_multi(keys)
 
 
+def update_timestamps_multi(entities, update_last_updated_time=True):
+    """Update the created_on and last_updated fields of all given entities.
+
+    Args:
+        entities: list(datastore_services.Model). List of model instances to
+            be stored.
+        update_last_updated_time: bool. Whether to update the
+            last_updated field of the model.
+    """
+    for entity in entities:
+        entity.update_timestamps(
+            update_last_updated_time=update_last_updated_time)
+
```

Comment:
We already have this function in the BaseModel right? CAn we use that instead?

Reply:
I have actually refactored the code so that the method in `BaseModel` calls this function.